xml/ha_storage_protection.xml

```diff
@@ -698,6 +698,66 @@ Timeout (msgwait)  : 10
    </procedure>
   </sect1>
 
+  <sect1 xml:id="sec.ha.storage.protect.diskless-sbd">
+   <title>Setting up Diskless SBD</title>
+   <para>
+    SBD can be operated in a diskless mode. As diskless SBD is not able
+    to handle split-brain scenarios in a two-node cluster, it is required
+    for this fencing mechanism to use a minimum of three nodes.
+   </para>
+   <para>
+    When setting up SBD, you need to take several timeout values into account.
+    For details, see <xref linkend="sec.ha.storage.protect.watchdog.timings"/>.
+   </para>
+   <para>
+    The following steps are necessary to set up diskless storage-based
+    protection:
+   </para>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>Keep in mind, we do not use a shared disk. As such, we do not
+       need the <varname>SBD_DEVICE</varname> entry in the configuration file.
+      </para>
+    </step>
+    <step>
+     <para>On each node, enable the SBD service to start at boot time: </para>
+     <screen>&prompt.root;<command>systemctl</command> enable sbd</screen>
+    </step>
+    <step>
+     <para>Restart the cluster stack on each node to ensure that the
+      <systemitem>sbd</systemitem> service is started:</para>
+     <screen>&prompt.root;<command>systemctl</command> stop pacemaker
+&prompt.root;<command>systemctl</command> start pacemaker</screen>
+     <para> This automatically triggers the start of the SBD daemon. </para>
+    </step>
+    <step>
+      <para>
+       If the cluster and SBD are correctly set up and running, check if
```

Comment:
Not sure if this should be the other way round: If <parameter>have-watchdog=true</parameter> has been set in the CIB,  the cluster has correctly detected SBD?